icons/shield_us_lht.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="20" height="20" viewBox="0 0 5.292 5.292" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
+ <circle style="fill:#fff;fill-opacity:1;stroke:#000;stroke-width:.264583;stroke-dasharray:none;stroke-opacity:1" cx="2.646" cy="2.646" r="2.514"/>
+ <path style="fill:#ddcba4;fill-opacity:1;stroke:none;stroke-width:.264583;stroke-dasharray:none;stroke-opacity:1" d="M 2.6458333,4.7624998 A 2.1166666,2.1166666 0 0 1 0.58622172,3.1339702 2.1166666,2.1166666 0 0 1 1.695875,0.75431094 2.1166666,2.1166666 0 0 1 4.2672939,1.2852662"/>
```

Comment:
> If there's a second flesh-colored shield component, we should standardize for real though.

`#ddcba4` is the California MUTCD’s standard “cream” color, also indicated as “buff” or “tan” in some specifications. It complements the brown in our color palette, as intended by Caltrans, but tan/cream/buff is such a rare color on shields that I’m OK with leaving it out of the palette until another case for it arises. We have other one-off deviations from the palette anyways.

/ref https://github.com/ZeLonewolf/openstreetmap-americana/pull/562#discussion_r991796731